{{cookiecutter.python_name}}/pyproject.toml

```diff
@@ -74,3 +74,7 @@ build_dir = "{{cookiecutter.python_name}}/labextension"
 
 [tool.jupyter-releaser.options]
 version_cmd = "hatch version"
+
+[tool.jupyter-releaser.hooks]
+before-build-npm = ["python -m pip install jupyterlab~=3.1", "jlpm"]
```

Comment:
Do we need to run the `build:prod` script as well? Or does `jlpm` already trigger the build?